doc/helpers/Input-extensions.md

```diff
@@ -0,0 +1,27 @@
+# InputExtensions Attached Properties
+Provides various attached properties for _input controls_, such as `TextBox` and `PasswordBox`.
+
+## Properties
+Property|Type|Description
+-|-|-
+AutoDismiss|bool|Whether the soft-keyboard will be dismissed when the enter key is pressed.
+AutoFocusNext|bool|Whether the focus will move to the next focusable element when the enter key is pressed.\*
```

Comment:
Should it start with "Is" ?

Reply:
As it's more of a feature than a property value that can be read, am ok leaving the Is out.